### PR TITLE
[Julia] Rename exercises

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

**Review by cmcaine — APPROVED** [review]
I haven't actually checked these changes properly. Probs fine, tho

**Comment by SaschaMann** [comment]
> I haven't actually checked these changes properly. Probs fine, tho

The main change is renaming the exercises from their concept name to an exercise name. `multiple-dispatch` to `encounters` and `unicode-identifiers` to `🕑🕤🕗`

**Comment by cmcaine** [comment]
I thought that was it, but reading diffs on my phone is hard. Sounds good
to me.

File: languages/julia/config.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
             },
             {
                 "uuid": "2b2b8bed-ff39-4e77-a2fe-254125b53a1c",
-                "slug": "unicode-identifiers",
+                "slug": "🕑🕤🕗",
```
Comment (ErikSchierboom) [inline languages/julia/config.json:49]: LOL

**Review by ErikSchierboom — APPROVED** [review]

**Review by iHiD — APPROVED** [review]

**Comment by SaschaMann** [comment]
(rebase merged to keep the changes readable. All the dead links turned it into a mess)

**Pull request merged**